### ISSUE TITLE
Misleading error message when attempting to read binary VTK files

### ISSUE THREAD

**Issue by lassoan** [nschloe/meshio#415]
Meshio seems to only support ascii VTK files, not binary. This would not be a problem, but the error that is printed does not give any clue that the issue is that the file is binary.

Example:

```
>>> mesh = meshio.read(r"c:\tmp\model-ascii.vtk")
>>> mesh = meshio.read(r"c:\tmp\model-bin.vtk")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2019-07-30\lib\Python\Lib\site-packages\meshio\_helpers.py", line 174, in read
    return format_to_reader[file_format].read(filename)
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2019-07-30\lib\Python\Lib\site-packages\meshio\_vtk.py", line 172, in read
    out = read_buffer(f)
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2019-07-30\lib\Python\Lib\site-packages\meshio\_vtk.py", line 206, in read_buffer
    _read_sub_section(f, info)
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2019-07-30\lib\Python\Lib\site-packages\meshio\_vtk.py", line 295, in _read_sub_section
    assert info.section == "FIELD", "Unknown section '{}'.".format(info.section)
AssertionError: Unknown section '
```

Example ascii and binary files are available here: https://1drv.ms/f/s!Arm_AFxB9yqHt_BH5m1ZSSfibgVrOg

**Comment by nschloe** [comment]
meshio _does_ support binary VTK, it's even the preferred format. I'll take a look at what's going wrong here next week.

**Comment by nschloe** [comment]
Fixed now, 3.0.6 released.

**Comment by lassoan** [comment]
Thank you very much!
